build/csv/csv-gen.sh

```diff
@@ -37,7 +37,14 @@ function generate_csv() {
 
     "${YQ_CMD_MERGE[@]}" "$CSV_FILE_NAME" "$ASSEMBLE_FILE_OCP"
 
+    # We don't need to include these files in csv as ocs-operator creates its own.
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/rook-ceph-operator-config_v1_configmap.yaml"
```

Comment:
If we don't remove these, it caused a failure downstream?

Reply:
this is was extra files created which are not required.